creation/probes.md

```diff
@@ -63,7 +63,7 @@ port 1500 via cloud-init.
             bootcmd:
               - setenforce 0
               - dnf install -y nmap-ncat
-              - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
+              - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\nContent-Length: 12\\n\\nHello World!'
```

Comment:
I'm guessing this is the same fix as this [PR](https://github.com/kubevirt/kubevirt/pull/3741)
can you add a clarification on the commit message?

Reply:
This is about providing a correct example to the user.

Sure, I can link that PR in the commit message, for completeness.

Reply:
Please resolve if you think the commit msg features all the info required.

Reply:
I don't have a resolve button, but consider this resolved :)